testsuite/helpers/TsCmp.h

```diff
@@ -61,10 +60,20 @@ namespace storage
 
 	};
 
-	TsCmpActiongraph(const Actiongraph::Impl& actiongraph, const Expected& expected);
+	/**
+	 * Loads "<name>-probed.xml", <name>-staging.xml" and
+	 * "<name>-expected.txt", compares the probed with the staging
+	 * devicegraph and verifies the resulting actiongraph. Additionally it
+	 * generates a picture of the actiongraph.
+	 */
+	TsCmpActiongraph(const string& name);
+
+	TsCmpActiongraph(const Actiongraph& actiongraph, const Expected& expected);
 
     private:
 
+	void doit(const Actiongraph& actiongraph, const Expected& expected);
```

Comment:
doit() is not a good name for anything... what does it do? Let's try to find a more speaking name.